flexmeasures/ui/templates/assets/asset_properties.html

```diff
@@ -245,152 +258,1077 @@ <h3>All child assets for {{ asset.name }}
+
+        const iconMap = {
+            "sensor": "fa fa-microchip",
+            "string": "fa fa-solid fa-lock",
+            "boolean": "fa fa-solid fa-flag",
+            "array": "fa fa-solid fa-list-ul",
+            "default": "fa fa-solid fa-question-circle",
+        }
+
+        const flexModelAllowedUnits = {
+            "power": ["MW", "kW"],
+            "energy": ["MWh", "kWh"],
+            "boolean": ["Boolean"],
+            "efficiency": ["%"],
+        }
+
+        const assetFlexModelSchema = {
+            "soc-min": null,
+            "soc-max": null,
+            "soc-minima": null,
+            "soc-maxima": null,
+            "soc-targets": null,
+            "soc-gain": [],
+            "soc-usage": [],
+            "roundtrip-efficiency": null,
+            "charging-efficiency": null,
+            "discharging-efficiency": null,
+            "storage-efficiency": null,
+            "prefer-charging-sooner": null,
+            "prefer-curtailing-later": null,
+            "power-capacity": null,
+            "consumption-capacity": null,
+            "production-capacity": null,
+        }
+
+        const FlexModelFieldValidTypes = {
+            "soc-min": typeThree,
+            "soc-max": typeThree,
+            "soc-minima": typeTwo,
+            "soc-maxima": typeTwo,
+            "soc-targets": typeTwo,
+            "soc-gain": typeFour,
+            "soc-usage": typeFour,
+            "roundtrip-efficiency": typeFive,
+            "charging-efficiency": typeThree,
+            "discharging-efficiency": typeThree,
+            "storage-efficiency": typeThree,
+            "prefer-charging-sooner": typeOne,
+            "prefer-curtailing-later": typeOne,
+            "power-capacity": typeThree,
+            "consumption-capacity": typeThree,
+            "production-capacity": typeThree,
+        };
+
+        const assetFlexModelFieldDesc = {
+            "soc-min": { "units": flexModelAllowedUnits.energy, "desc": "A constant lower boundary for all values in the schedule (defaults to 0).", "types": "Fixed value or Sensor." },
+            "soc-max": { "units": flexModelAllowedUnits.energy, "desc": "A constant upper boundary for all values in the schedule (defaults to max soc target, if provided).", "types": "Fixed value or Sensor." },
+            "soc-minima": { "units": flexModelAllowedUnits.energy, "desc": "Set points that form lower boundaries, e.g. to target a full car battery in the morning (defaults to NaN values).", "types": "Fixed value or Sensor." },
+            "soc-maxima": { "units": flexModelAllowedUnits.energy, "desc": "Set points that form upper boundaries at certain times (defaults to NaN values).", "types": "Fixed value or Sensor." },
+            "soc-targets": { "units": flexModelAllowedUnits.energy, "desc": "Exact set point(s) that the scheduler needs to realize (defaults to NaN values).", "types": "Fixed value or Sensor." },
```

Comment:
Can this say "One fixed value or a dynamic signal (via a sensor)." instead of "Fixed value or Sensor."?

I believe this explains better what we enable here.

(in the end, we should use these wordings on the flex context as well, in a later PR)